benchmarks/benchmarks/traj_reader.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import division, absolute_import, print_function
+
+try:
+    from MDAnalysis.coordinates.DCD import DCDReader
+    from MDAnalysis.coordinates.XTC import XTCReader
+    from MDAnalysis.coordinates.TRR import TRRReader
+    from MDAnalysisTests.datafiles import XTC, TRR, DCD
+except ImportError:
+    pass
+
+class TrajReaderBench(object):
+    """Benchmarks for trajectory file format reading."""
+    params = (['XTC', 'TRR', 'DCD'])
+    param_names = ['traj_format']
+
+    def setup(self, traj_format):
+        self.traj_dict = {'XTC': [XTC, XTCReader],
+                          'TRR': [TRR, TRRReader],
+                          'DCD': [DCD, DCDReader]}
+        self.traj_file, self.traj_reader = self.traj_dict[traj_format]
+
+    def time_reads(self, traj_format):
+        """Simple benchmark for reading traj file formats
+        from our standard test files.
+        """
+        self.traj_reader(self.traj_file)
```

Comment:
Creation only -- I don't know if the readers differ in the amount of data read in on initialization.

If creation time is not of interest or there's a standard number of frames that all reader test files have in them, the test could be modified to stride through the test files over n frames as well. If initialization time is not of particular interest, then that could be moved to the `setup`. I was mostly just aiming to demonstrate a basic parametrized benchmark test.